From 97fad5f32bfe24b3a3ce8b3923b27099b746de06 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Wed, 6 Mar 2013 13:41:23 +0100
Subject: [PATCH 4/4] Fix various usage mistakes
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

quota: Fix long variant for -A option in usage
repquota: Fix long variant for -c option in usage and manual
repquota: Fix long variant for -C option in usage and manual
repquota: Add -a, --all options to usage

Signed-off-by: Petr Písař <ppisar@redhat.com>
Signed-off-by: Jan Kara <jack@suse.cz>
---
 quota.c    | 2 +-
 repquota.8 | 4 ++--
 repquota.c | 5 +++--
 3 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/quota.c b/quota.c
index 56e71a8..1befbde 100644
--- a/quota.c
+++ b/quota.c
@@ -102,7 +102,7 @@ static void usage(void)
 -i, --no-autofs           do not query autofs mountpoints\n\
 -F, --format=formatname   display quota of a specific format\n\
 -f, --filesystem-list     display quota information only for given filesystems\n\
--A, --nfs-all             display quota for all NFS mountpoints\n\
+-A, --all-nfs             display quota for all NFS mountpoints\n\
 -m, --no-mixed-pathnames  trim leading slashes from NFSv4 mountpoints\n\
     --show-mntpoint       show mount point of the file system in output\n\
     --hide-device         do not show file system device in output\n\
diff --git a/repquota.8 b/repquota.8
index a71cf6b..bb6d7ab 100644
--- a/repquota.8
+++ b/repquota.8
@@ -84,11 +84,11 @@ to be read-write with quotas.
 Report all quotas, even if there is no usage. Be also more verbose about quotafile
 information.
 .TP
-.B -c, --batch-translation
+.B -c, --cache
 Cache entries to report and translate uids/gids to names in big chunks by scanning
 all users (default). This is good (fast) behaviour when using /etc/passwd file.
 .TP
-.B -C, --no-batch-translation
+.B -C, --no-cache
 Translate individual entries. This is faster when you have users stored in database.
 .TP
 .B -t, --truncate-names
diff --git a/repquota.c b/repquota.c
index 43de31d..50683b9 100644
--- a/repquota.c
+++ b/repquota.c
@@ -56,9 +56,10 @@ static void usage(void)
 -p, --raw-grace             print grace time in seconds since epoch\n\
 -n, --no-names              do not translate uid/gid to name\n\
 -i, --no-autofs             avoid autofs mountpoints\n\
--c, --batch-translation     translate big number of ids at once\n\
--C, --no-batch-translation  translate ids one by one\n\
+-c, --cache                 translate big number of ids at once\n\
+-C, --no-cache              translate ids one by one\n\
 -F, --format=formatname     report information for specific format\n\
+-a, --all                   report information for all mount points with quotas\n\
 -h, --help                  display this help message and exit\n\
 -V, --version               display version information and exit\n\n"), progname);
 	fprintf(stderr, _("Bugs to %s\n"), MY_EMAIL);
-- 
1.8.1.4

